### PR TITLE
Use JSON imports instead of PDFJSDev.json

### LINKED ISSUE #18563: Use JSON imports rather than a custom Babel transform
The Babel plugin has a custom JSON transform that replaces `PDFJSDev.json("$ROOT/some/path.json")` with the contents of `some/path.json`:
https://github.com/mozilla/pdf.js/blob/c60c0d1c6df3a4e93da9d9ad4186eb07e837e024/external/builder/babel-plugin-pdfjs-preprocessor.mjs#L43-L55

This is only used once, in the Chromium extension code:
https://github.com/mozilla/pdf.js/blob/c60c0d1c6df3a4e93da9d9ad4186eb07e837e024/web/chromecom.js#L197-L199

There is a Stage 3 proposal that would allow directly importing JSON files:
```js
import i18nFileAccessLabels from "./chrome-i18n-allow-access-to-file-urls.json" with { type: "json" };
```

This feature is already supported by default by Node.js, Chrome, Safari and Webpack. Given that the code where we use `PDFJSDev.json` is bundled with webpack, this syntax is safe to use and we can remove the custom `PDFJSDev.json` feature.

However, ESLint doesn't support this syntax yet because it's only a Stage 3 proposal and not part of the standard yet (it will hopefully become part of the standard in October). There are two options:
- wait until the first ESLint minor release after October, which will support this syntax by default
- use `@babel/eslint-parser` to let ESLint parse this syntax

### PR DESCRIPTION
**Pull request by nicolo-ribaudo**
JSON imports are now supported by all tools used in PDF.js' build process. The `chromecom.js` file is bundled by webpack and import attributes are thus removed, so browser compatibility for this new syntax is not relevant.

Fixes #18563

### CODE REVIEW

**Comment by Snuffleupagus** [comment]
Sorry, but this doesn't seem to work (at all) when running `gulp chromium` (or even `gulp generic`) locally and a number of the GitHub Actions work-flows failed as well.

**Comment by nicolo-ribaudo** [comment]
Keeping it as draft because, even if ESLint 9.14 addes support for `with` (https://github.com/eslint/eslint/releases/tag/v9.14.0) and we are using 9.15, it still complains about it :thinking:

**Comment by Snuffleupagus** [comment]
> Keeping it as draft because, even if ESLint 9.14 addes support for `with` (https://github.com/eslint/eslint/releases/tag/v9.14.0) and we are using 9.15, it still complains about it 🤔

Could that be solved by updating the version here? https://github.com/mozilla/pdf.js/blob/658a0a16c15dff3e7d6c08222aee64b4d64166b3/eslint.config.mjs#L80

File: external/builder/fixtures_babel/evals.js [hunk 1]
```diff
@@ -5,7 +5,6 @@ var d = PDFJSDev.test('FALSE');
 var e = PDFJSDev.eval('TRUE');
 var f = PDFJSDev.eval('TEXT');
 var g = PDFJSDev.eval('OBJ');
-var h = PDFJSDev.json('$ROOT/external/builder/fixtures_babel/evals.json');
```
Comment (Snuffleupagus) [inline external/builder/fixtures_babel/evals.js:8]: Does this makes that file unused?
If so, we should also remove `external/builder/fixtures_babel/evals.json` as part of this patch.

File: web/chromecom.js [hunk 2]
```diff
@@ -194,14 +197,10 @@ function requestAccessToLocalFile(fileUrl, overlayManager, callback) {
...
     // Use Chrome's definition of UI language instead of PDF.js's #lang=...,
     // because the shown string should match the UI at chrome://extensions.
-    // These strings are from chrome/app/resources/generated_resources_*.xtb.
-    const i18nFileAccessLabel = PDFJSDev.json(
-      "$ROOT/web/chrome-i18n-allow-access-to-file-urls.json"
-    )[chrome.i18n.getUILanguage?.()];
-
-    if (i18nFileAccessLabel) {
+    const lang = chrome.i18n.getUILanguage?.();
+    if (lang && i18nFileAccessLabels[lang]) {
```
Comment (Snuffleupagus) [inline web/chromecom.js:201]: Can't we reduce the amount of changes necessary here?
```suggestion
    const i18nFileAccessLabel = i18nFileAccessLabels[chrome.i18n.getUILanguage?.()];
    if (i18nFileAccessLabel) {
```

File: web/chromecom.js [hunk 3]
```diff
@@ -194,14 +197,10 @@ function requestAccessToLocalFile(fileUrl, overlayManager, callback) {
...
       document.getElementById("chrome-file-access-label").textContent =
-        i18nFileAccessLabel;
+        i18nFileAccessLabels[lang];
```
Comment (Snuffleupagus) [inline web/chromecom.js:203]:
```suggestion
        i18nFileAccessLabel
```

**Review by Snuffleupagus — COMMENTED** [review]
Please fix the typo in the commit message: instad -> instead

**Review by Snuffleupagus — APPROVED** [review]
r=me, thank you.

**Pull request merged**